src/build.py

```diff
@@ -650,8 +651,14 @@ def SyncRepos(filter, sync_lkgr=False):
 
   good_hashes = None
   if sync_lkgr:
-    lkgr_file = os.path.join(WORK_DIR, 'lkgr')
-    cloud.Download('lkgr', lkgr_file)
+    if sys.platform.startswith('linux'):
+      buildername = 'linux'
+    elif sys.platform == 'darwin':
+      buildername = 'mac'
+    elif sys.platform == 'win32':
```

Comment:
I'd add `else: assert "unexpected sys.platform"` for an easier to parse error message if this ever doesn't work